src/app/results/results.component.html

```diff
@@ -75,8 +75,8 @@
                 <ul class="pagination" id="pag-bar">
                     <li class="page-item"><span class="page-link" href="#" (click)="decPresentPage()">Previous</span></li>
                     <li class="page-item" *ngFor="let num of getNumber(maxPage)"><span class="page-link" *ngIf="presentPage>=4 && num<=noOfPages" [class.active_page]="getStyle(presentPage-3+num)"
-                            (click)="getPresentPage(presentPage-3+num)" href="#">{{presentPage-3+num}}</span>
-                        <span class="page-link" *ngIf="presentPage<4 && num<=noOfPages" [class.active_page]="num + 1 == presentPage" (click)="getPresentPage(num)"
+                            (click)="getPresentPage(presentPage-4+num)" href="#">{{presentPage-3+num}}</span>
```

Comment:
**convert** `{{getPresentPage(presentPage-4+num)}}` => `{{getPresentPage(presentPage-3+num)}}`
**logic**:- let us assume presentPage is 4 and num is 0 => presentPage-4+num = 0 => `this.presentPage-1` in below function is  -1. correct me if I am wrong.